Python/PRP/Region/pyConvexIsect.cpp

```diff
@@ -57,18 +59,73 @@ PyMethodDef pyConvexIsect_Methods[] = {
     PY_METHOD_TERMINATOR
 };
 
+PY_GETSET_GETTER_DECL(ConvexIsect, planes)
+{
+    PyObject* list = PyTuple_New(self->fThis->getPlanes().size());
+    for (size_t i = 0; i < self->fThis->getPlanes().size(); i++)
+        PyTuple_SET_ITEM(list, i, pySinglePlane_FromSinglePlane(&(self->fThis->getPlanes()[i])));
+    return list;
+}
+
+PY_PROPERTY_SETTER_MSG(ConvexIsect, planes, "To add planes, use addPlane")
```

Comment:
`addPlane` should probably be changed to accept a `plConvexIset::SinglePlane` if the expected arguments of position, normal are not supplied.